examples/slash_commands/option_choice.py

```diff
@@ -0,0 +1,43 @@
+from disnake import Option, OptionChoice, Interaction, Intents
+from disnake.ext import commands
+from disnake.ext.commands import slash_command
+
+import random
+
+intents = Intents.all()
+# In production please choose the intents coresponding to your use case.
+
+# We create the instance of the bot.
+bot = commands.Bot(intents=intents)
+
+# For this example we're going to make a rock-paper-scissors game
+# which will use OptionChoice as the input.
+@slash_command(name='rps', description='Play a game of rock-paper-scissors with the bot.',
+    options=[
+        Option(name='choice', description='Select your choice for rock-paper-scissors.', choices=[
+            OptionChoice(name='Rock', value='rock'),
+            OptionChoice(name='Paper', value='paper'),
+            OptionChoice(name='Scissors', value='scissors')
```

Comment:
`[OptionChoice(name=..., value=...), ...]` works, but it's not the shortest way to define option choices. You can use a dict instead:
```python
choices={
    'Rock': 'rock',
    'Paper': 'paper',
    'Scissors': 'scissors'
}
```

Reply:
I decided to go for OptionChoice as this is an OptionChoice example, it doesn't necessarily have to be short but detailed and easy to understand

Reply:
Okay, fair enough